<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpotifyApp</class>
 <widget class="QMainWindow" name="SpotifyApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpotifyApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lstPlaylist">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtBuscar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblBuscar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar MÃºsica:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstResultadosBusca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdicionar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>530</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adicionar a Playlist &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTocarPlaylist">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Tocar Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblResultados">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlaylist">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalvarPlaylist">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>540</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpaPlaylist">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemover">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>482</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover da Playlist</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
